library/ssl_tls.c

```diff
@@ -693,6 +693,25 @@ int mbedtls_ssl_derive_keys( mbedtls_ssl_context *ssl )
         return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
     }
 
+#if defined(MBEDTLS_SSL_CID)
+    /* Copy own and peer's CID if the use of the CID
```

Comment:
Note (on the commit, not this particular line): this is likely to conflict with the work being done in #2618, so at some point we'll probably have to decide on a merge order between those PRs. (Nothing to change here, just noting.)